extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/run-remote-queries.test.ts

```diff
@@ -0,0 +1,103 @@
+import { expect } from 'chai';
+import { parseResponse } from '../../../remote-queries/run-remote-query';
+
+describe('run-remote-queries', () => {
```

Comment:
Thanks for adding the test 🙌🏽  
Very minor - could we change the name of this test (and the file name itself) to `run-remote-query`? Just for consistency with the name of the file we're testing 😸